### PR TITLE
Fix deploy: path

### PR DESCRIPTION
**Pull request by rachelcarmena**

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ jobs:
       run: |
         git checkout master
         bash ./generate-haddock-docs.sh docs/wip
```
Comment (calvellido) [inline .github/workflows/deploy.yml]:
```suggestion
        bash ./generate-haddock-docs.sh docs
```

Like this

Comment (rachelcarmena) [reply .github/workflows/deploy.yml]: Good idea!! :+1: However, now I wonder if there will be conflicts with the previous content of `docs`. What do you think about it?

Comment (rachelcarmena) [reply .github/workflows/deploy.yml]: NVM, `docs/haddock` is removed as a first step in the script :+1: Thanks!

Comment (rachelcarmena) [reply .github/workflows/deploy.yml]: Fixed! Thanks @calvellido !!

**Pull request merged**